### ISSUE TITLE
Is l8w8jwt affected by CVE-2024-49195?

### ISSUE THREAD

**Issue by nodet** [GlitchedPolygons/l8w8jwt#49]
I wonder whether l8w8jwt (we're using from 2.3.2 to 2.3.4) is affected by [CVE-2024-49195](https://www.cve.org/CVERecord?id=CVE-2024-49195), which is described at https://mbed-tls.readthedocs.io/en/latest/security-advisories/mbedtls-security-advisory-2024-10-1/.

Would you please advise?
Thanks.

**Comment by GlitchedPolygons** [comment]
@nodet Thank you for notifying me about this! Yes, it's affected. I will release an update asap 👍🏻

**Comment by GlitchedPolygons** [comment]
@nodet done! Thanks again for pointing this out :)

Fixed in [release 2.4.0](https://github.com/GlitchedPolygons/l8w8jwt/releases/tag/2.4.0)

Please consider [donating to the project 🙂 ](https://glitchedpolygons.com/donate)

Take care, and good luck with your projects!

**Comment by GlitchedPolygons** [comment]
@nodet 
If the main underlying crypto library of any of my projects has a bug that is classified with "high severity", I am automatically affected. Always. 😉 

But you're right, l8w8jwt itself doesn't call that function. I replied off the top of my head (I recognized the function name and explicitly remember using it), but it's actually [the l8w8jwt GUI](https://github.com/GlitchedPolygons/l8w8jwtgui) that makes use of that function, not the base lib.

I'll have to update that as well, for safety's sake :)
